source/gfoidl.Base64/ThrowHelper.cs

```diff
@@ -1,8 +1,17 @@
 ﻿using System;
 using System.Buffers;
 using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
 using System.Resources;
 
+#if NETSTANDARD2_0
+namespace System.Diagnostics.CodeAnalysis
+{
+    [AttributeUsage(AttributeTargets.Method, Inherited = false)]
+    public sealed class DoesNotReturnAttribute : Attribute { }
```

Comment:
```suggestion
    internal sealed class DoesNotReturnAttribute : Attribute { }
```
The public visibility is generating conflicts like 
`Warning	CS0436	The type 'DoesNotReturnAttribute' in 'C:\dev\Jwt\src\JsonWebToken\DoesNotReturnAttribute.cs' conflicts with the imported type 'DoesNotReturnAttribute' in 'gfoidl.Base64, Version=1.1.765.0, Culture=neutral, PublicKeyToken=null'. Using the type defined in 'C:\dev\Jwt\src\JsonWebToken\DoesNotReturnAttribute.cs'.`

Reply:
Perfect, thanks 👍